src/server/server.ts

```diff
@@ -226,23 +229,23 @@ export class GraphQLTransportWSServer {
               // result is a GraphQL ExecutionResult, which has an optional errors property
               params.callback = (error: any, result: any) => {
                 if (!error) {
-                  this.sendMessage(connection, connectionContext, reqId, MessageTypes.GQL_DATA, result);
+                  this.sendMessage(connectionContext, reqId, MessageTypes.GQL_DATA, result);
                 } else if (error.errors) {
```

Comment:
yeah it's returning an ExecutionResult in the promise resolve so it is was what result contains 😄